files/base.rb

```diff
@@ -222,6 +223,10 @@ def handler_settings
     settings[settings_key]
   end
 
+  def reset_api_settings!
+    @settings['api'].merge!(settings['api_client']) if settings['api_client']
+    settings['api']
```

Comment:
this return value isn't used anywhere except specs. since method is supposed to be mutating a variable - can you spec for that instead?